packages/corvid-cli/src/utils/bi.js

```diff
@@ -35,8 +35,15 @@ function sendBiEvent(evid) {
   };
 }
 
+function getBiContextHeader(isVisible) {
+  const data = { builderEnv: "local", isHeadless: !isVisible };
+  const encoded = new Buffer(JSON.stringify(data)).toString("base64");
```

Comment:
1. When running in `node v10.15` I get this warning:
```events.js:189 (node:4912) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.```
Using - `Buffer.from(JSON.stringify(data)).toString('base64')` works
2. Please verify with @antonpod that this is the expected output

Reply:
Done.
Verified with @antonpod. It's ok.